docs/Installation.md

```diff
@@ -39,43 +46,25 @@ included in your source repo might be too old to properly setup the container).
 After that, just run:
 
 ```shell
-$ sudo docker-compose -f docker-latest.yml --env-file config.env up -d
+$ cd picatrix/docker
+$ sudo docker-compose -f docker-latest.yml up -d
 ```
 
 That will download the latest build and deploy the picatrix docker container.
-To be able to connect to picatrix in a jupyter shell, run:
-
-```shell
-$ sudo docker logs docker_picatrix_1
-...
-    To access the notebook, open this file in a browser:
-        file:///home/picatrix/.local/share/jupyter/runtime/nbserver-6-open.html
-    Or copy and paste one of these URLs:
-        http://....:8899/?token=...
-     or http://127.0.0.1:8899/?token=...
-...
-```
+To be able to connect to picatrix, open the following URL in a browser
+window:
+[http://localhost:8899/?token=picatrix](http://localhost:8899/?token=picatrix)`
 
-On Windows you can also use the Docker Desktop client and click the running
-Docker container to get the log window open and copy the URL from there.
-
-Copy the URL that starts with `http://127.0.0.....` and paste it into a browser
-window and you should have a fully working copy of Jupyter running, with an
+You should have a fully working copy of Jupyter running, with an
 active picatrix library.
 
-Also remember that all notebooks you create inside the container that are
-not part of the `data` folder, that is that are created inside the container
-itself will be deleted once the container is upgraded. It is therefore
-recommended to create all notebooks that you wish to store inside the `data`
-folder of the container (which is mapped to a folder on the host).
-
 ### Upgrade Container
 
 To upgrade the container using the latest build, you can run:
 
 ```shell
-$ sudo docker-compose -f docker-latest.yml --env-file config.env pull
-$ sudo docker-compose -f docker-latest.yml --env-file config.env up -d
+$ sudo docker-compose -f docker-latest.yml pull
```

Comment:
do you. have multiple docker-compose files? If not you can just name it docker-compose.yml and then you can do:
```
sudo docker-compose up -d
```

It will default to docker-compose.yml

Reply:
I've got two

docker-compose.yml is the one that I use for building new images...  that is that one does not use the pre-built image.

I might also just rename the docker compose file , or create a `build` directory